mozilla/blurts-server
# MNTOR-1800: Part 3 - unsub emails endpoint and generate link util function

# References:

Jira: MNTOR-1800



# Description
* Util function to generate unsubscribe URL
* API endpoint to verify unsub url and unsub the user from monthly monitor reports

# How to test
* Use the function `generateUnsubscribeLink` to generate a link
* Try visit the url (GET request) and the user should be unsubscribed to the monitor report

# Checklist (Definition of Done)
